addons/mail/static/src/rtc/call_action_list.js

```diff
@@ -51,6 +69,21 @@ export class CallActionList extends Component {
         }
     }
 
+    onClickDownloadLogs() {
+        const data = window.JSON.stringify(Object.fromEntries(this.rtc.state.logs));
+        const blob = new window.Blob([data], { type: "application/json" });
+        const url = window.URL.createObjectURL(blob);
+        const a = document.createElement("a");
```

Comment:
I would rather go for something a little more explicit for the variable name, such as "downloadLink".

Reply:
Same as above